openedx/openedx-learning
# feat: Copy object tags [FC-0062]

This PR adds:

- A function `copy_tags` in `api.py` to copy tags between objects.
- A new field `is_copied` in `ObjectTag`

## Supporting information

- Github issue: https://github.com/openedx/modular-learning/issues/177
- It's used in: https://github.com/openedx/edx-platform/pull/35596
- Internal ticket: [FAL-3616](https://tasks.opencraft.com/browse/FAL-3616)

## Testing instructions

- Verify that the tests cover the new code.
- Follow the testing instructions in https://github.com/openedx/edx-platform/pull/35596

## Deadline

ASAP
